octokit/pagination.py

```diff
@@ -0,0 +1,38 @@
+from .resources import Resource
+
+class Pagination(object):
+  def __init__(self, *args, **kwargs):
+    # TODO (howei): possibly extract auto_paginate from kwargs
+    # so users can do client = Client(auto_paginate=True)
+    self.auto_paginate = False
+    self.last_response = None
```

Comment:
oops no idea why I did that. I was going to put that in ratelimit.py instead of client.py, since client doesn't directly use it and ratelimit seems to be the only thing that does. thoughts?